app/models/user.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable new-cap */
```

Comment:
If we need to add this line in every entity maybe we should disable the rule directly on the eslint config file